openwisp_radius/api/views.py

```diff
@@ -84,13 +85,33 @@
 
 
 class FreeradiusApiAuthentication(BaseAuthentication):
+    def _check_client_ip_and_return(self, request, uuid):
+        client_ip, _is_routable = get_client_ip(request)
+        try:
+            ip_addresses = OrganizationRadiusSettings.objects.get(
+                organization__pk=uuid
+            ).freeradius_allowed_hosts
+            allowed_ip_list = ip_addresses.split(',') if ip_addresses else None
```

Comment:
why not create a model method (even with a `@property`) which returns the right value ready for usage?

Untested hint:

```python
@property
def freeradius_allowed_hosts_list(self):
    addresses = self.freeradius_allowed_hosts
    if not addresses:
        return app_settings.FREERADIUS_ALLOWED_HOSTS
    return self.freeradius_allowed_hosts.split(',')
```